aspect_rules_py 1.11.7 fail()s in host_platform_repo on Windows. The
platform_libc string_flag default is loaded for every configured target,
so rust/proto analysis dies even when python/ packages are deleted.

--- a/uv/private/host/repository.bzl
+++ b/uv/private/host/repository.bzl
@@ -55,9 +55,10 @@
         else:
             fail("Unknown libc from ldd --version %r" % res.stdout)
 
-    # TODO: Windows
-    # TODO: Other
+    elif os == "windows":
+        return "msvc", "19.0"
 
+    # TODO: Other
     fail("Unsupported platform {}".format(os))
 
 def _host_platform_repo_impl(rctx):
